Makefile

```diff
@@ -16,7 +16,7 @@ setup: ## setup development dependencies
 .PHONY: setup
 
 lint: ## run the fast go linters
-	golangci-lint run
+	./bin/golangci-lint run
```

Comment:
why this one?

Reply:
Yeah this doesn't actually affect the build, but I noticed that if you use `make setup` to install the dependencies it will put them in `./bin`, so this won't work.